pkg/ccl/streamingccl/streamproducer/replication_stream_test.go

```diff
@@ -198,21 +198,21 @@ func TestReplicationStreamInitialization(t *testing.T) {
 
 	t.Run("failed-after-timeout", func(t *testing.T) {
 		h.SysDB.Exec(t, "SET CLUSTER SETTING stream_replication.job_liveness_timeout = '1s'")
-		rows := h.SysDB.QueryStr(t, "SELECT crdb_internal.init_stream($1)", h.Tenant.ID.ToUint64())
+		rows := h.SysDB.QueryStr(t, "SELECT crdb_internal.start_replication_stream($1)", h.Tenant.ID.ToUint64())
 		jobID := rows[0][0]
 
-		time.Sleep(500 * time.Millisecond) // Wait for the job status transitioning
+		//time.Sleep(500 * time.Millisecond) // Wait for the job status transitioning
```

Comment:
Drop this comment?
Is succeedssoonduration of 2 seconds enough? Even when running under `-race`?